src/core/mentions/index.ts

```diff
@@ -211,3 +137,193 @@ async function getWorkspaceProblems(cwd: string): Promise<string> {
 	}
 	return result
 }
+
+// Handler implementations
+const urlHandler: HandlerConfig = {
+	name: "url",
+	test: (mention: string) => mention.startsWith("http"),
+	handler: async (mention, { urlContentFetcher, launchBrowserError }) => {
+		const tag = createXmlTag("url_content", { url: mention })
+		let content: string
+
+		if (launchBrowserError) {
+			content = handleError(launchBrowserError, "fetching content")
+		} else {
+			try {
+				content = await urlContentFetcher.urlToMarkdown(mention)
+			} catch (error) {
+				content = handleError(error, `fetching content for ${mention}`)
+			}
+		}
+		return wrapContent(content, tag)
+	},
+}
+
+const fileHandler: HandlerConfig = {
+	name: "file",
+	test: (mention: string, { osInfo }) => (osInfo !== "win32" ? mention.startsWith("/") : mention.startsWith("\\")),
+	handler: async (mention, { cwd, osInfo }) => {
+		let mentionPath = mention.slice(1)
+		const isFolder = mention.endsWith("/")
```

Comment:
Is this different on windows?

Reply:
yes, it different how to escape space on window  
oh my isFolder function not update, let me update it